tls/tls_internal.h

```diff
@@ -168,7 +168,29 @@ struct tls_handshake_t {
 	};
 	union {
 		unsigned char		premaster[TTLS_PREMASTER_SIZE];
-		unsigned char		tmp[TTLS_HS_RBUF_SZ];
+		struct {
+			union {
+				struct {
+					unsigned short cs_cur_len;
+				};
```

Comment:
the wrapping unnamed `struct` seems unnecessary

Reply:
Removed unnamed struct here, as it's not needed.